src/dcqc/updaters.py

```diff
@@ -33,10 +33,8 @@ def update(self, suites: List[SuiteABC]):
                 collapsed_dict[url] = "RED"
             elif "AMBER" in statuses:
                 collapsed_dict[url] = "AMBER"
-            elif "GREEN" in statuses:
-                collapsed_dict[url] = "GREEN"
             else:
-                collapsed_dict[url] = "NONE"
+                collapsed_dict[url] = "GREEN"
```

Comment:
Would this potentially lead to false positives?

Reply:
I think that could only happen if there were more statuses added

Reply:
Does it make sense to keep the None then?

Reply:
Yeah, I guess so. Better safe than sorry. I'll add a test for that case as well